src/components/Calculator.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable max-classes-per-file */
+import React from 'react';
+import '../App.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please don't disable eslint rules. This is one of the basic requirements.

Reply:
since the `Table` component doesn't have any state, it's better to use a PureComponent instead to increase performance

Read more about it [here](https://stackoverflow.com/questions/41340697/react-component-vs-react-purecomponent)

```js
class Table extends React.PureComponent {

...

}
```